web/rootfs/defaults/meet.conf

```diff
@@ -32,3 +32,13 @@ location /http-bind {
     proxy_set_header X-Forwarded-For $remote_addr;
     proxy_set_header Host {{ .Env.XMPP_DOMAIN }};
 }
+
+{{ if .Env.ETHERPAD_URL_BASE }}
+# Etherpad-lite
+location /etherpad/ {
```

Comment:
I think we need some config here to allow for cross-origin requests. This is what I get otherwise:

<img width="1276" alt="Screen Shot 2019-06-20 at 20 50 46" src="https://user-images.githubusercontent.com/317464/59873824-20be4600-939d-11e9-9766-3177032117cb.png">

Reply:
I could not find what raising it.
Upgrade connection for websocket added in 5f06ecc

Reply:
Ack